### ISSUE TITLE
Error code -79

### ISSUE THREAD

**Issue by ashimtamang00** [foxlet/bootra1n#47]
Error code -79 
Even after enabling safe mode

**Comment by fegato** [comment]
I think we need a Bootra1n version with Checkra1n 0.10.0 instead of 0.10.1

**Comment by zakusa** [comment]
found any solution yet ? i keep getting error 79

**Comment by demhademha** [comment]
What USB type are you usin?

**Comment by zakusa** [comment]
not sure ( some type of apple usb ) . i tried different usb ports ( 3.0, 3.1, 2.0) in my computer and all give same error. i have ipad air 2 and i find that many people with ipad air 2 is getting the same problem. if anyone has ipad air 2 and found a solution, it would be great if you told me how to solve it.

**Comment by Pjink91** [comment]
I’ve done it, try like houndred times, i got Windows 10 i had same error on 0.10.1 but it passed nad jailbraked my air 2!

**Comment by zakusa** [comment]
Do you have iOS 13.4.1 or 13.4?

**Comment by fegato** [comment]
so you have to keep tryin? windows 10 you mean using bootra1n?

**Comment by Pjink91** [comment]
Yes yes try many times , different ports nad cables, i leaved it for a
month nad yesterday i was like „last try” and it worked

wt., 28 kwi 2020 o 14:06 fegato <notifications@github.com> napisał(a):

**Comment by fegato** [comment]
nad what exactly was your last try set ? :)

**Comment by Pjink91** [comment]
And* sorry but different languages on keyboard make some mess.. Maybe try different type of pendrive, i think i was lucky, on my amd laptop nothing was working before... it was posible to run only bootrain, i left my air 2 turned off for like a month i think, last night-couple beers + 420 and then i was like „last try on fuc.... Windows” . It was like all other air 2, bootra1n started to upload magic into my iPad but after few Lines it stopped, same shit... but after about 1 minute magic continues and voila! Done and jailbreaked :) but.. i can install cydia but i cant download anything on cydia... it crashes all the time so watch out because you cant install safe shutdown :(

**Comment by fegato** [comment]
thankyou for your answer! if I can't manage to borrow a mac I'll have some beers and try again

**Comment by Pjink91** [comment]
Forgot to say. iOS 13.4 i got, wish you good luck!

**Comment by zakusa** [comment]
that explains it. i saw that everyone got error 79 on ios 13.4.1 for ipad air 2 but i saw some people actually jailbreak ios 13.4 with ipad air 2

**Comment by bggbhgh** [comment]
Same here I’ve enabled safe mode and removed passcode still coming up with error -79

**Comment by Pjink91** [comment]
I did it only with V ticked on options, doing allways in verbose mode dont
know why heh, it took me a month of doing the same thing in bootra1n... So
it was more than 100 times.... After all another problem appeared, cydia
crashes everytime you want to install antthing so no "safe shutdown "
option, have to watch battery all the timw

sob., 2 maj 2020, 00:20 użytkownik bggbhgh <notifications@github.com>
napisał:

**Comment by zakusa** [comment]
did anyone find a solution yet for ipad air 2 ios 13.4.1?

**Comment by hoco21** [comment]
I'm getting the same error code. I have the proper apple cable, I'm plugged into a USB 3.0 but I've also tried 2.0 (and the ports on the back of my computer). I'm running IOS 12.4.7 on an iPhone 6. I've tried almost all th settings in both the boot menu and the settings in the application. I have no password on my device at the moment either. 

Is there any way to fix this?
